### DISCUSSION TITLE
Can't access RTSP on Wyze 3 cams

### DISCUSSION THREAD

**Discussion by Z3r0shin** [gtxaspec/wz_mini_hacks#484] [Q&A]
Hello!

I've been scratching my head all evening trying to understand what I am doing wrong.
I'm able to access the web server for both cams and make the changes there, but I can't open the rtsp feed through VLC.

Here is a screenshot of the web server :
![Screenshot 2023-02-14 210104](https://user-images.githubusercontent.com/63606612/218908790-8f2ba4c6-0b4e-44cc-95f9-5135bae441b4.png)

Is there something that could block the feed somewhere?
I have an old v1 cam with the old beta rtsp firmware that I can access no problem, with several IPCam doing the same.

**Comment by vic4news** [comment]
Try just using unicast instead of video1_unicast

**Reply by Z3r0shin** [reply]
I tried that too, nothing.
I also disabled hd stream and used only the sd stream, and vice versa, still nothing.

**Reply by jcoulter** [reply]
I am in a similar boat although I noticed that I can get audio but not video through the video1_unicast

**Comment by jonbeckman** [comment]
What firmware version is your v3 on? I have two cams and had both of them working with RTSP. I then upgraded to v4.36.9.139 on one and now it doesn't work. The README is ambiguous as it first suggests <v4.36.9.13, but then suggests <=v4.36.9.13.
> Works on ANY firmware release up to 4.36.9.139 (DO NOT UPGRADE BEYOND THIS FW...

So it may be that?

**Reply by Z3r0shin** [reply]
Yeah... the firmware is 4.36.9.139... Dammit really?! That's really not cool.

Is there a reason why beyond that firmware it is not possible to RTSP the cams? Or has it just not cracked yet?

**Reply by jcoulter** [reply]
I have gotten the high quality stream to play on 4.36.9.139 successfully. The low quality stream will not play and life is just easier if I disable it and use the unicast url. The web server configuration is amazing.  I was unable to get them to work with the rtsp firmware.

**Comment by Bob-AOL** [comment]
The instructions below actually work. The configuration provided is appropriate for each type of stream. Stick with port 554 just because. VLC should grab any of the MRLs given an appropriately configured stream.

Firmware version 4.36.9.139 is the last version that is compatible with wz_mini. The upgraded release has been recalled by Wyze due to major problems leaving 4.36.9.139 as the current and recommended version. You can avoid having a newer upgrade pushed at your camera and breaking it by disabling upgrades in the wz_mini.conf file. The RTSP Beta firmware also works with wz_mini and will prohibit unintended upgrades, though it does bring its own idiosyncrasies to the party which need to be understood when using it.

![RTSP READ](https://user-images.githubusercontent.com/110629141/220678862-5a383940-fd80-491f-b6ea-967853a6b66f.jpg)

![RTSP CONFIG](https://user-images.githubusercontent.com/110629141/220678942-c0778aa9-7cd4-4853-8997-3c8f7529198f.jpg)
